src/routes/PlayQuiz/index.tsx

```diff
@@ -171,47 +285,109 @@ export default function PlayQuiz() {
-            selectedOptionId={
-              selectedOptionsData.find((x) => x.questionId === selectedQuestion.questionId)?.selectedOptionId
-            }
-          />
-        )}
-        <div className={classNames(styles.scoreContainer, 'ml-lg')}>
+      {quizInfo.name && (
+        <Title order={2} mb="xl" pb="xl">
+          Play game for {quizInfo.name}
+        </Title>
+      )}
+      <PanelGroup autoSaveId="playQuizPanel" direction="horizontal">
+        <Panel defaultSize={20} minSize={20}>
+          <Accordion multiple defaultValue={quizInfo.categories.map((x) => `${x.categoryId}`)} variant="separated">
+            {quizInfo.categories.map((category) => (
+              <Accordion.Item key={category.categoryId} value={`${category.categoryId}`}>
+                <Accordion.Control>
+                  <Title order={6}>{category.categoryName}</Title>
+                </Accordion.Control>
+                <Accordion.Panel>
+                  <Button.Group orientation="vertical">
+                    {category.questions.map((question) => (
+                      <Button
+                        leftIcon={getTeamAvatar(question)}
+                        my="sm"
+                        radius="md"
+                        disabled={
+                          !winner &&
+                          !attemptedQuestionIds.includes(question.questionId) &&
+                          (!isGameStarted ||
+                            (!!selectedQuestion && selectedQuestion?.questionId !== question.questionId) ||
+                            (!selectedQuestion && !isPlaying))
+                        }
+                        variant={selectedQuestion?.questionId === question.questionId ? 'filled' : 'light'}
+                        color={getQuestionColor(question)}
+                        fullWidth
+                        onClick={() => showQuestion(question.questionId, category.categoryId)}
+                        key={question.questionId}>
+                        {isQuestionPointsHidden ? (
+                          <Text>Question {question.questionNum}</Text>
+                        ) : (
+                          <Group position="apart">
+                            <Text>Question {question.questionNum}</Text>
+                            <Badge variant="filled" color="red" size="sm">
+                              Points {question.points}
+                            </Badge>
+                          </Group>
+                        )}
+                      </Button>
+                    ))}
+                  </Button.Group>
+                </Accordion.Panel>
+              </Accordion.Item>
+            ))}
+          </Accordion>
+        </Panel>
+        <ResizeHandle />
+        <Panel defaultSize={60} maxSize={60}>
+          {winner
+            ? !selectedQuestion && (
+                <Title pt="xl" mt="xl" color="grey" align="center" order={3}>
+                  {getWinnerMessage()}
```

Comment:
Winner message is not shown in a modal but where the question renders?

Reply:
we show if in modal when game completed. But if a user open completed game or refreshes the page after game completes. we shows this message where the question renders (when no question selected).

Reply:
Ok